fabtests/prov/efa/src/efa_shared.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: BSD-2-Clause OR GPL-2.0-only */
```

Comment:
This is useful for general efa specific fabtests in the future. So I intend to keep it in a separate .h